envs/upkie_base_env.py

```diff
@@ -95,13 +95,13 @@ def reset(
         self,
         *,
         seed: Optional[int] = None,
-        return_info: bool = False,
+        return_info: bool = True,
         options: Optional[dict] = None,
     ) -> Union[np.ndarray, Tuple[np.ndarray, Dict]]:
         """!
         Resets the spine and get an initial observation.
 
-        @param seed Will be used once we upgrade to gym >= 0.21.0.
+        @param seed It is not used yet but should be (TODO).
```

Comment:
Feel free to open an issue :+1: 

Also it's better to use the `TODO(user)` syntax in the code to make sure there is someone responsible for doing it. If there is nobody else you can assign this to me: `TODO(scaron): ...`.